test/fixtures/SpokePool.Fixture.ts

```diff
@@ -222,24 +231,25 @@ export function getRelayHash(
   return { relayHash, relayData };
 }
 
-export function getDepositParams(
-  _recipient: string,
-  _originToken: string,
-  _amount: BigNumber,
-  _destinationChainId: number,
-  _relayerFeePct: BigNumber,
-  _quoteTime: BigNumber,
-  _maxCount?: BigNumber
-): string[] {
+export function getDepositParams(args: {
+  recipient?: string;
+  originToken: string;
+  amount: BigNumber;
+  destinationChainId: number;
+  relayerFeePct: BigNumber;
+  quoteTimestamp: number;
+  message?: string;
+  maxCount?: BigNumber;
+}): string[] {
   return [
-    _recipient,
-    _originToken,
-    _amount.toString(),
-    _destinationChainId.toString(),
-    _relayerFeePct.toString(),
-    _quoteTime.toString(),
-    "0x",
-    _maxCount ? _maxCount.toString() : consts.maxUint256.toString(),
+    args.recipient ?? randomAddress(),
+    args.originToken,
+    args.amount.toString(),
+    args.destinationChainId.toString(),
+    args.relayerFeePct.toString(),
+    args.quoteTimestamp.toString(),
+    args.message ?? "0x",
+    args?.maxCount?.toString() ?? consts.maxUint256.toString(),
   ];
```

Comment:
This was the entry point for this change - we need to be able to generate deposits with a custom message.